### PR TITLE
Basic views

### PR DESCRIPTION
**Pull request by stshayna**
@DenzelBraithwaite Created basic view pages for items:
- index
- show
- new/create
- _form.html.erb (for creating a new item)

Added "name" as a new field in Items table too.

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+  end
+
+  private
+
+  def find_item
+    @item = Item.find(params[:id])
+  end
+
+  def item_params
+    params.require(:item).permit(:description, :expiration_date, :status, :type)
```
Comment (DenzelBraithwaite) [inline app/controllers/items_controller.rb:31]: change type to item_type

File: app/models/item.rb [hunk 2]
```diff
@@ -15,12 +16,11 @@ class Item < ApplicationRecord
   belongs_to :user
   has_and_belongs_to_many :allergens
   has_and_belongs_to_many :diets
-  validates_presence_of :user_id, :description, :expiration_date, :type, :status
+  validates_presence_of :user_id, :description, :expiration_date, :type, :status, :name
```
Comment (DenzelBraithwaite) [inline app/models/item.rb:19]: I changed type to item type in my branch, but if you do it as well here, there won't be any merge conflicts when I push my seed branch.

File: app/views/items/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-6 shadow">
+      <%= simple_form_for @item do |f| %>
+        <%= f.input :name %>
+        <%= f.select :type, collection: Item::TYPES %>
```
Comment (DenzelBraithwaite) [inline app/views/items/_form.html.erb:6]: item_type

**Review by DenzelBraithwaite — CHANGES_REQUESTED** [review]
Looks good, just need to change type to item_type in a few files then we can merge.

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (stshayna) [reply app/controllers/items_controller.rb]: Updated the field name in the Items table!

File: app/models/item.rb (hunk 2 above)
Comment (stshayna) [reply app/models/item.rb]: Updated the field name in the Items table!

File: app/views/items/_form.html.erb (hunk 3 above)
Comment (stshayna) [reply app/views/items/_form.html.erb]: Updated! Refer to changes in latest commits. Thanks!

**Review by DenzelBraithwaite — APPROVED** [review]
Perfect

**Pull request merged**